appveyor.yml

```diff
@@ -0,0 +1,11 @@
+environment:
+  matrix:
+    - TOXENV: py38
+
+build: off
+
+install:
+  - pip install tox==3.13.2
+
+test_script:
+  - tox --force-dep="pytest @ git+https://github.com/dougthor42/pytest.git@win"
```

Comment:
Can you blueyed/master again.

Reply:
done.

Reply:
Wait, do you mean `@my-master`?

Reply:
Oh, sure.
Though I've thought about using "master" instead, and probably will - but then I'll fix it in one go.